src/main/java/com/ericsson/ei/config/ConfigurationLogger.java

```diff
@@ -57,8 +57,8 @@ private void logConfiguration() {
                 + "threads.corePoolSize: " + environment.getProperty("threads.corePoolSize") + "\n"
                 + "threads.queueCapacity: " + environment.getProperty("threads.queueCapacity") + "\n"
                 + "threads.maxPoolSize: " + environment.getProperty("threads.maxPoolSize") + "\n"
-                + "failed.notification.collection-name: " + environment.getProperty("failed.notification.collection-name") + "\n"
-                + "failed.notification.database-name: " + environment.getProperty("failed.notification.database-name") + "\n"
+                + "missedNotificationCollectionName: " + environment.getProperty("missedNotificationCollectionName") + "\n"
+                + "missedNotificationDataBaseName: " + environment.getProperty("missedNotificationDataBaseName") + "\n"
```

Comment:
I don't know what happened here in the rebase, but on master we want to have the name change to failed notifications right?

Reply:
done.